homeassistant/components/overseerr/coordinator.py

```diff
@@ -0,0 +1,59 @@
+"""Overseer coordinator s."""
+from datetime import timedelta
+import logging
+from typing import Self
+
+from overseerr_api import ApiClient, RequestApi
+from overseerr_api.models import RequestCountGet200Response
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CannotConnect(HomeAssistantError):
+    """Unable to connect to the web site."""
+
+
+class OverseerrRequestData:
+    """Keep data for Overseerr entities."""
+
+    def __init__(self, api_client: ApiClient, hass: HomeAssistant) -> None:
+        """Initialise the weather entity data."""
```

Comment:
<strike>weather</weather>

Reply:
Fixed! Took some inspiration from the weather integration and forgot to remove all traces :)